### DISCUSSION TITLE
Inquiry About CUDA 118 Support in Version 0.6.2 Release

### DISCUSSION THREAD

**Discussion by KLOSYX** [vllm-project/vllm#8934] [Q&A]
I recently noticed that the 0.6.2 version release does not include a Python package that supports CUDA 118. I went through the release notes but couldn't find any mention of this. Could someone please clarify if the new version does not support CUDA 118?

**Comment by tarekabouzeid** [comment]
I think the release pipeline failed, that's why the CUDA 118 built was not generated, you can check [here](https://github.com/vllm-project/vllm/actions/runs/11041663331/job/30672417507)

**Comment by Hsun1128** [comment]
Is there a chance that version v0.6.2 could be released with CUDA 11.8 or 12.1 support? Versions above v0.6 have fixed many issues with the "tools".

**Reply by tarekabouzeid** [reply]
Yesterday @simon-mo  ran the pipeline for [CU 11.8](https://buildkite.com/vllm/release/builds/1176#01926e51-5732-4d36-8f31-cae51d3b1d23). 

you can download that artifact from [here](https://vllm-wheels.s3.amazonaws.com/7193774b1ff8603ad5bf4598e5efba0d9a39b436/vllm-0.6.3.dev0%2Bg7193774b.d20241008.cu118-cp38-abi3-manylinux1_x86_64.whl)

**Comment by simon-mo** [comment]
Here's the URL, the version number is a bit messed up sorry:

https://vllm-wheels.s3.us-west-2.amazonaws.com/7193774b1ff8603ad5bf4598e5efba0d9a39b436/vllm-0.6.3.dev0%2Bg7193774b.d20241008.cu118-cp38-abi3-manylinux1_x86_64.whl

**Reply by s0h3yl** [reply]
_Referencing this [open ticket](https://github.com/vllm-project/vllm/issues/9021)._

Thank you, @simon-mo - much appreciated!

Are there plans [to fix the CI](https://github.com/vllm-project/vllm/actions/workflows/publish.yml) or re-release them for `0.6.2`, as `0.6.3` wasn't released yet?
